doc/kci_rootfs.md

```diff
@@ -0,0 +1,156 @@
+### How to build a rootfs image using kci_rootfs
+
+We will be using `kernelci/staging-debos` docker image for this purpose.
+
+1. Pull the docker image `docker pull kernelci/staging-debos`
+
+2. Clone the kernelci-core repo.
+
+    ```
+    git clone https://github.com/kernelci/kernelci-core.git 
+    ```
+3. Start the docker and get into it.
+
+   ```
+   sudo docker run -itd -v $(pwd)/kernelci-core:/kernelci-core --device /dev/kvm --privileged kernelci/staging-debos
+   sudo docker exec -it <container_id> bash
+   cd /kernelci-core/ 
+   ```
+4. Now to check if everything works, type `./kci_rootfs --help` it should produce below help message.
+
+    ```
+    usage: kci_rootfs [-h] [--yaml-configs YAML_CONFIGS]
+
+                      {validate,list_configs,list_variants,build,upload} ...
+
+    optional arguments:
+
+      -h, --help            show this help message and exit
+
+      --yaml-configs YAML_CONFIGS
+
+                            Path to the YAML configs file
+    Commands:
+
+      {validate,list_configs,list_variants,build,upload}
+
+                            List of available commands
+
+        validate            Validate the rootfs YAML configuration
+
+        list_configs        List all rootfs config names
+
+        list_variants       List all rootfs variants
+
+        build               Build a rootfs image
+
+        upload              Upload a rootfs image
+    ```
+5. To list available rootfs configuration, we can use `list_configs` option.
```

Comment:
we VS you. I see that you use "we" always, shouldn't be better use "you"?
(Note: I am not English native)

Reply:
+1 for "you"

Reply:
`we` Vs `you` is mostly personal choice/taste. I feel "we" usage as way of guiding community user (let's try this stuff together..) and "you" more like non-guided/commanding approach (you do this step & next step is..) .  If you still want to go with `you`, i'll update the patch.

Reply:
I think "we" can also be seen as "we, the kernelci folks who know about this stuff" and put some distance with new people coming to the project.  If there is a "we" it means there are "others" who are not part of that group, so I find it more exclusive than inclusive.  Some communities probably want to have a strong group identity as it's a natural feeling, but that's for social reasons which are imho very different to documenting how a command line tool works.

The "you" pronoun can be used in an impersonal way, so it's very neutral.  See this definition for example:
https://chambers.co.uk/search/?query=you&title=21st

> 2 any or every person • You don't often see that nowadays.

It's similar to using "one" but less formal, i.e. "one can use the `list_configs` option".

Reply:
Someone with similar view: https://ux.stackexchange.com/questions/57305/we-vs-you-in-documentation Anyway, it takes less time to update the patch one way or another rather than concluding which is correct approach, PR updated.